components/RegistrationForm/index.js

```diff
@@ -60,21 +104,24 @@ const RegistrationForm = () => {
                 isEditing = true
             }
             register(isEditing, 'attendee', registrationData)
+            alert("Successfully registered!");
         }
 
         setValidated(true);
+        
 
-        /*if (form.fileUpload.value != '') {
+        if (form.fileUpload.value != '') {

```

Comment:
This code needs more testing, I had commented it out because I needed to figure out what the file control had in its default state.